src/utils.ts

```diff
@@ -65,3 +69,25 @@ export function getRefillsAllowed(
     return null; // dispense doesn'r haee a "refills allowed" component
   }
 }
+
+// TODO does this need to null-check
+export function getMedication(
+  resource: MedicationRequest | MedicationDispense
+): Medication {
+  return {
+    medicationReference: resource.medicationReference,
+    medicationCodeableConcept: resource.medicationCodeableConcept,
+  };
+}
+
+export function getPrescriptionDetailsEndpoint(encounterUuid: string) {
```

Comment:
extracted this out to a function because I needed to generate the same key in the mutate function. (see above)